src/TableView/TableView.jsx

```diff
@@ -72,12 +80,28 @@ class TableView extends Component {
     }
   }
 
+  componentWillUpdate(nextProps) {
+    const {errorMessage} = nextProps;
+
+    if (errorMessage) {
+      this.toaster.show({
+        message: errorMessage,
+        className: 'pt-intent-danger',
+        timeout: 0,
+        onDismiss: () => {}
+      });
+    }
+  }
+
   componentWillUnmount() {
+    this.toaster.getToasts().forEach(toast => this.toaster.dismiss(toast.key));
     this.props.clearData();
   }
 
-  handleDeleteData = id => {
-    this.props.deleteData(this.state.activeSchema.url, id, this.state.activeSchema.plural);
+  handleDeleteData = item => {
+    if (window.confirm(`Delete ${item.name}?`)) { // eslint-disable-line no-alert
```

Comment:
There is a component which can be used for confirmation: http://blueprintjs.com/docs/#components.overlay